pkg/bytecode/code.go

```diff
@@ -32,6 +32,12 @@ const (
 	OpTrue
 	// OpFalse represents the boolean literal false.
 	OpFalse
+	// OpBang represents the bang unary operator, which performs logical
+	// negation on a boolean operand.
+	OpBang
```

Comment:
maybe this should be called `OpNot` ? no strong feelings, just and idea, seems closer to the actual operation than bang.

Reply:
Good idea, I like that name better